firmware/Core/Src/telecommands/telecommand_definitions.c

```diff
@@ -363,6 +364,30 @@ const TCMD_TelecommandDefinition_t TCMD_telecommand_definitions[] = {
         .readiness_level = TCMD_READINESS_LEVEL_FOR_OPERATION
     },
     // ****************** END: MPI_telecommand_definitions ********************
+    // ****************** START SECTION: flash_bank_telecommand_defs ******************
+
+    {
+        .tcmd_name = "flash_bank_read",
+        .tcmd_func = TCMDEXEC_Flash_Bank_Read,
+        .number_of_args = 1,
+        .readiness_level = TCMD_READINESS_LEVEL_FLIGHT_TESTING,
```

Comment:
If these telecommands aren't meant to be used in flight, can you please set an appropriate level for them here?

Reply:
Moved to ground usage only. Thanks for the catch